gradio/components/annotated_image.py

```diff
@@ -15,8 +15,6 @@
 
 set_documentation_group("component")
 
-PIL.Image.init()  # fixes https://github.com/gradio-app/gradio/issues/2843
```

Comment:
We may need to keep this one as we don't import image_utils here

Reply:
There's no way to import `gradio.components.annotated_image` without the parent module `gradio.components` getting imported, which imports `gradio.components.image` and then `image_utils`.